cmd/grafana-agentctl/Dockerfile.windows

```diff
@@ -7,7 +7,7 @@ WORKDIR /src/agent
 
 SHELL ["cmd", "/S", "/C"]
 
-RUN ""C:\Program Files\git\bin\bash.exe"" -c 'RELEASE_BUILD=${RELEASE_BUILD} VERSION=${VERSION} make agentctl'
+RUN ""C:\Program Files\git\bin\bash.exe" -c 'RELEASE_BUILD=${RELEASE_BUILD} VERSION=${VERSION} make agentctl"
```

Comment:
This doesn't look right, there's a mismatched `'` now. Did you mean 

```
RUN ""C:\Program Files\git\bin\bash.exe" -c 'RELEASE_BUILD=${RELEASE_BUILD} VERSION=${VERSION} make agentctl'"
```

Reply:
so close even though it looks and turns out is wrong. It is the only 1 that runs through make agent just fine but does give a bad return code in the end unfortunately. It just failed locally. Every reasonable quote option is wrong for windows cmd/docker/bash so I'll set it back to the working powershell 1 second